alpine-base-php/php8/etc/php8/conf.d/00_json.ini

```diff
@@ -0,0 +1 @@
+extension=json.so
```

Comment:
This is now always included in PHP: https://php.watch/versions/8.0/ext-json

For info:

> PHP Warning:  PHP Startup: Unable to load dynamic library 'json.so' (tried: /usr/lib/php8/modules/json.so (Error loading shared library /usr/lib/php8/modules/json.so: No such file or directory), /usr/lib/php8/modules/json.so.so (Error loading shared library /usr/lib/php8/modules/json.so.so: No such file or directory)) in Unknown on line 0